loader/icd_dispatch_generated.c

```diff
@@ -96,12 +98,81 @@ CL_API_ENTRY cl_int CL_API_CALL clGetDeviceIDs(
             num_devices);
 #endif // defined(CL_ENABLE_LAYERS)
     KHR_ICD_VALIDATE_HANDLE_RETURN_ERROR(platform, CL_INVALID_PLATFORM);
-    return platform->dispatch->clGetDeviceIDs(
```

Comment:
This is a generated file. You don't want to modify this, you want to modify the generator.

Reply:
sorry.  I closed this PR by mistake and I created PR 153. Please review.
I updated the icd_dispatch_generated.c file.